core/src/main/java/org/apache/calcite/rel/hint/HintStrategyTable.java

```diff
@@ -48,15 +52,26 @@
 
   /** Empty strategies. */
   // Need to replace the EMPTY with DEFAULT if we have any hint implementations.
-  public static final HintStrategyTable EMPTY = new HintStrategyTable(new HashMap<>());
+  public static final HintStrategyTable EMPTY = new HintStrategyTable(
+      new HashMap<>(), new HashMap<>(), HintErrorLogger.INSTANCE);
```

Comment:
`new HashMap<>()` -> `Collections.emptyMap()` may be better? I think `EMPTY`  here implicate that it's an immutable object.

Reply:
I agree, thanks.